asyncstdlib/_core.py

```diff
@@ -1,6 +1,6 @@
+from __future__ import annotations
```

Comment:
This sadly doesn't work on Python 3.6, which I would like to still support unless some serious changes are necessary. Please explicitly quote types if needed instead.